library/std/tests/run-time-detect.rs

```diff
@@ -6,7 +6,6 @@
         all(target_arch = "aarch64", any(target_os = "linux", target_os = "android")),
         all(target_arch = "powerpc", target_os = "linux"),
         all(target_arch = "powerpc64", target_os = "linux"),
-        any(target_arch = "x86", target_arch = "x86_64"),
```

Comment:
Should this be removed for aarch64 as well? You might need to put this behind `cfg(bootstrap)` so the stage0 compiler still accepts it.